edx_proctoring/templates/proctored_exam/ready_to_resume.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div class="sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      Your exam is ready to be resumed.
```

Comment:
This template is almost exactly the same as `edx_proctoring/templates/proctored_exam/entrance.html`. I debated whether we should just use the the existing entrance template in place of this one, but I thought it was valuable to indicate to the learner that the exam is being resumed and that they are not starting from scratch.

Reply:
I agree with this decision. Duplication in this case is healthy in my opinion. It provides better flexibility for the future and separation of concerns.